relayer/chains/sui/listener.go

```diff
@@ -31,9 +31,44 @@ func (p *Provider) Listener(ctx context.Context, lastProcessedTx relayertypes.La
 		txInfo.TxDigest = p.cfg.StartTxDigest
 	}
 
+	if txInfo.TxDigest == "" {
```

Comment:
Is lastProcessed tx digest is unused now?

Reply:
Not ignored, it is still considered